### ISSUE TITLE
doc: update gh-pages

### ISSUE THREAD

**Issue by edkerk** [SysBioChalmers/yeast-GEM#259]
### Description of the issue:
Currently https://sysbiochalmers.github.io/yeast-GEM/ is build from the `gh-pages` branch, which is not automatically updated when `master` is changed. The page mainly gives the information from `README.md`, but if this file is updated on `master` (which happens with each new release), the version on gh-pages is outdated.

#### Expected feature/value/output:
- Bare minimum: a page with up-to-date information. Could for instance be arranged by writing an Action that merges `master` to `gh-pages` after each merge to `master`.
- Alternative: a completely different, simpler landing page. GitHub-savvy people would be confident with the usual https://github.com/SysBioChalmers/yeast-GEM/ page, perhaps https://sysbiochalmers.github.io/yeast-GEM/ should be more accessible to those less familiar with GitHub. Just a few lines introducing the model and citation, then links to (i) latest release + list of all releases; (ii) memote report of latest release + history report; (iii) Issues page, to report problems; (iv) the main repo page for more detailed information as given in the `README.md`? This page doesn't even have to be updated with new releases, just link to for instance https://github.com/SysBioChalmers/yeast-GEM/releases/latest/. 

#### Current feature/value/output:
Not up-to-date, representing the same README as in the repo landing page.

**Comment by BenjaSanchez** [comment]
> Alternative: a completely different, simpler landing page

I like this idea! As you mention, it would simplify the releasing process & make for a newbie-friendly landing page. Maybe some link to a Git/Github tutorial, to encourage people to clone the repo instead of downloading?

**Comment by edkerk** [comment]
> Maybe some link to a Git/Github tutorial, to encourage people to clone the repo instead of downloading?
Great idea!

**Comment by edkerk** [comment]
A proposed new page is prepared on [gh-pagesNew](https://github.com/SysBioChalmers/yeast-GEM/tree/gh-pagesNew), the result is shown https://sysbiochalmers.github.io/yeast-GEM/.

Any feedback welcome! Note that the link to the memote report (on the left) does not work at the moment, but once 8.5.0 is released it will show a memote report for the latest published release.

**Comment by BenjaSanchez** [comment]
@edkerk I went through the new page and I think it contains everything required for installation/usage/etc, without being too wordy. Nice work! My only comment is that next to "Latest memote report" you could have a link that says something like "What is memote?" and points to the publication, as I imagine newcomers to the field might not know what memote is.

**Comment by edkerk** [comment]
Now implemented.
